pyperformance/data-files/benchmarks/bm_gc_collect/run_benchmark.py

```diff
@@ -0,0 +1,64 @@
+import pyperf
+import gc
+
+CYCLES = 100
+LINKS = 20
+
+class Node:
+    def __init__(self):
+        self.next = None
+        self.prev = None
+    
+    def link_next(self, next):
+        self.next = next
```

Comment:
If you add `next.back = self` here you don't need the `link_back` method.

Reply:
Done